extensions/resource-deployment/src/services/notebookService.ts

```diff
@@ -54,6 +55,25 @@ export class NotebookService implements INotebookService {
 		});
 	}
 
+	/**
+	 * Inserts cell code given by {@param cellStatements} in an existing notebook given by {@param notebook} filepath at the location
+	 * {@param insertionPosition} and then launches the edited notebook.
+	 *
+	 * @param notebook - the path to notebook that needs to be launched
+	 * @param cellStatements - array of statments to be insterted in a cell
+	 * @param insertionPosition - the position at which cells are inserted. Default is a new cell at the beginning of the notebook.
+	 */
+	public async insertCellAndLaunchNotebook(notebook: string, cellStatements: string[], insertionPosition: number = 0): Promise<void> {
+		const notebook_1 = await this.launchNotebook(notebook);
+		notebook_1.edit((editBuilder: azdata.nb.NotebookEditorEdit) => {
+			// 5 is the position after the 'Set variables' cell in the deployment notebooks
```

Comment:
This comment seems like it belongs in deployClusterWizard instead

Reply:
good catching - fixing